test/functions/vass/setter-passed-to-another-fun.good

```diff
@@ -6,7 +6,7 @@ int by inout     true
 int by out       true
 int by ref       true
 int by const in  false
-int by const ref true
```

Comment:
I believe this change brings it to compliance with the rules for ref-pairs in the spec.